### PR TITLE
Readd fog density on 1.17

### PR DESCRIPTION
**Pull request by IMS212**

### CODE REVIEW

File: src/main/java/net/coderbot/iris/mixin/statelisteners/MixinGlStateManager.java [hunk 1]
```diff
@@ -36,8 +36,8 @@
 	}
 
 	static {
```
Comment (TehcJS) [inline src/main/java/net/coderbot/iris/mixin/statelisteners/MixinGlStateManager.java:38]: just delete the mixin

File: src/main/java/net/coderbot/iris/uniforms/CameraUniforms.java [hunk 2]
```diff
@@ -27,7 +27,7 @@ public static void addCameraUniforms(UniformHolder uniforms, FrameUpdateNotifier
 			.uniform3d(PER_FRAME, "previousCameraPosition", new PreviousCameraPosition(notifier));
 	}
 
-	private static int getRenderDistanceInBlocks() {
+	public static int getRenderDistanceInBlocks() {
 		return client.options.viewDistance * 16;
```
Comment (TehcJS) [inline src/main/java/net/coderbot/iris/uniforms/CameraUniforms.java:31]:
```diff
- return client.options.viewDistance * 16;
+ return client.gameRenderer.getViewDistance();
```
Use `GameRenderer#getViewDistance`

**Review by TehcJS — CHANGES_REQUESTED** [review]
:bike::house:

File: src/main/java/net/coderbot/iris/uniforms/CameraUniforms.java (hunk 2 above)
Comment (IMS212) [reply src/main/java/net/coderbot/iris/uniforms/CameraUniforms.java:31]: this should be something for another pr, as this is the current behavior of CameraUniforms.

**Comment by IMS212** [comment]
This doesn't seem to work properly, closing for now.

**Pull request closed without merging**